.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[unstable]
+features = ["host_dep"]
```

Comment:
I'm not sure about this change. It kind of changes things locally in a kind of "invisible" way. Would it perhaps make sense to add this as a note to the README instead?